app/screens/index.tsx

```diff
@@ -28,7 +28,7 @@ const withGestures = (Screen: React.ComponentType, styles: StyleProp<ViewStyle>)
 };
 
 const withIntl = (Screen: React.ComponentType) => {
-    return function IntlEnabledComponent(props: any) {
+    return function IntlEnabledComponent(props: never) {
```

Comment:
Shouldn't this be `unknonw` instead of `never`?

Reply:
unknown cause too many problems, that why I went with never.

Reply:
My only concern is that it is not idiomatic. Never is used for things that can never happen, like the return value of a function that always throws an error. But I am ok if it is used because of practical reasons.

Reply:
I can change it back to any in that case.... 0/5